src/components/ListItem.jsx

```diff
@@ -1,5 +1,59 @@
 import './ListItem.css';
+import { useState, useEffect } from 'react';
+import { updateItem } from '../api';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ item, listToken }) {
+	const one_day_in_ms = 24 * 60 * 60 * 1000;
+	// const one_day_in_ms = 60 * 2 * 1000; // 120 seconds for testing the reset timeframe
+	const currentDate = new Date();
+	const currentTimeInMilliseconds = Math.floor(currentDate.getTime());
+	const dateLastPurchasedInMilliseconds =
+		item && item.dateLastPurchased
+			? item.dateLastPurchased.seconds * 1000
+			: null;
+	const timeElapsed =
```

Comment:
Thanks for the review Lindsey. I see what you're saying about the timeElapsed being a const and therefore only changing once. I think Nadia and I had discussed whether or not we need the updates to happen live in real time, or just on page load. I would also be curious to hear what Nick has to say as well. 

Also, I was a little nervous putting the timeElapsed in the useEffect because I thought perhaps that would cause another infinite loop since the timeElapsed would always be changing. Maybe I just don't put it in the dependency array??